### PR TITLE
Contrail R4.0 support for ocata

### PR DESCRIPTION
**Pull request by alexey-mr**

### CODE REVIEW

File: manifests/network/contrail/analytics.pp [hunk 1]
```diff
@@ -199,7 +194,6 @@
   $auth_host                  = hiera('contrail::auth_host'),
   $auth_port                  = hiera('contrail::auth_port'),
   $auth_protocol              = hiera('contrail::auth_protocol'),
-  $auth_port_ssl              = hiera('contrail::auth_port_ssl'),
```
Comment (ankitchadha) [inline manifests/network/contrail/analytics.pp:202]: Did the variable change in Ocata/KS?

**Review by ankitchadha — APPROVED** [review]

File: manifests/network/contrail/analytics.pp (hunk 1 above)
Comment (alexey-mr) [reply manifests/network/contrail/analytics.pp:202]: yes, there is appropriate change in the tripleo heat templates (that is based on https://review.openstack.org/#/c/475048/21)

**Comment by alexey-mr** [comment]
@andrey-mp

**Comment by Andrey-mp** [comment]
@rrugge

**Pull request merged**